gradle/java9plus.gradle

```diff
@@ -0,0 +1,166 @@
+import org.gradle.build.DefaultJavaInstallation
+
+import org.gradle.jvm.toolchain.internal.JavaInstallationProbe
+
+if (testJavaHome) {
+
+    def testJavaInstallation = new DefaultJavaInstallation()
+    gradle.services.get(JavaInstallationProbe)
+        .checkJdk(new File(testJavaHome))
+        .configure(testJavaInstallation)
+    def currentJavaVersion = testJavaInstallation.javaVersion
+
+    if (isSupportedJavaVersion(currentJavaVersion)) {
+
+        tasks.withType(org.gradle.testing.IntegrationTest) {
+            ext.excludedTests = [
```

Comment:
:x: This should be a simple `def` in order not to hold the map

Reply:
done